lib/voyager-message/src/call.rs

```diff
@@ -86,11 +86,11 @@ pub struct FetchBlocks {
 ///
 /// # Implementation Note
 ///
-/// The returned [`Op`] ***MUST*** resolve to an [`OrderedHeaders`] data.
+/// The returned [`Op`] ***MUST*** resolve to an [`crate::data::OrderedHeaders`] data.
```

Comment:
in general, prefer to put only the type name in the link name, and use the full path in the link. it makes the generated docs a lot less noisy

```suggestion
/// The returned [`Op`] ***MUST*** resolve to an [`OrderedHeaders`][crate::data::OrderedHeaders] data.
```